ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -292,65 +292,48 @@ spec:
         - name: source
           workspace: pipeline
           subPath: src
-        - name: credentials
+        - name: dockerconfig
           workspace: registry-credentials
-        - name: cacert
+        - name: sslcertdir
           workspace: registry-cacert
 
-    # Index image contains a record of bundle images from which
-    # manifests could be extract in order to install an operator.
-    - name: generate-index
+    - name: build-fbc-scratch-catalog
       runAfter:
         - build-bundle
-        - check-ocp-cluster-version
```

Comment:
The removal of this causes that `get-supported-versions` is no longer ahead of this task, but it uses the results. I think we should check `check-ocp-cluster-version` before building index.

Reply:
Thanks, I reverted the condition.